frontend/src/app/app.module.ts

```diff
@@ -8,17 +8,33 @@ import { AngularFireModule } from '@angular/fire';
 import { AngularFirestoreModule } from '@angular/fire/firestore';
 import { environment } from '../environments/environment';
 import { ImagesComponent } from './images/images.component';
+import { ClarityModule } from '@clr/angular';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './home/home.component';
+import { ImagesModule } from './images/images.module';
+
+const appRoutes: Routes = [
+  { path: 'images', component: ImagesComponent },
+  { path: '', component: HomeComponent }
+];
 
 @NgModule({
   declarations: [
     AppComponent,
-    ImagesComponent
```

Comment:
good call. always a good idea to only declare components in a single module. And even better to break features up into feature modules. _EVEN MORE BETTERER_ if you follow `@angular/material`'s self contained component modules. This _allegedly_ can allow for better tree shaking. But we dont need to really worry about that sort of a thing for a small app like this one yet.